src/window.rs

```diff
@@ -1282,6 +1304,13 @@ pub enum Theme {
     Dark,
 }
 
+impl Default for Theme {
+    #[inline]
+    fn default() -> Self {
+        Self::Light
+    }
+}
+
```

Comment:
I know that while most users use a light theme, I don't actually think a default makes sense here?

Reply:
I fixed this here.
[fix: remove unnecessary default for theme](https://github.com/rust-windowing/winit/pull/2441/commits/42c7318ca004d8bdda7a73cb3a5cf31fb92836fb)